### DISCUSSION TITLE
Usage Guide

### DISCUSSION THREAD

**Discussion by amzon-ex** [roovo/obsidian-card-board#112] [Q&A] [answered]
Can someone help me to use this plugin properly?

I have created a tag board, filtered by some tags, named the columns by those tags and have created a markdown file that has a few tasks and uses those tags. No tasks show up on my board. Why is this happening?
![image](https://user-images.githubusercontent.com/45428344/204120506-21fc8a3d-33e8-4bc2-8690-730c2ef48c36.png)

**Comment by roovo** [comment, marked as answer]
Hiya - not sure what the issue could be....  I have just created a board as per your screenshot and added the attached [tasks.md](https://github.com/roovo/obsidian-card-board/files/10098209/tasks.md) to my vault and can see "task two" in the "General" column and "task three" in the "Academic" column.  Could you add the file to your vault and see if it works for you?

Hoping this helps!

**Comment by amzon-ex** [comment]
Oh I see now... The tags must be placed after every single task in the list? Is there no way to group all `#tasks/gen` tasks using the tag only once, for instance?

**Reply by roovo** [reply]
Hiya - the only way you can do this at the moment is if you put all tasks which you want to apply the same tag to in the same file and set the tag in the front matter as any tags in the front matter will be applied to all tasks in the document.  So if you have the following at the top of a document:

```
---
tags: [ aTag ]
---
```

Then all tasks in that doc will inherit the tag `#aTag`.

**Reply by amzon-ex** [reply]
Ah now I see it. Doing it the standard way (putting "hash"tags at the beginning of the document) does not work, but putting them in the front-matter works! Thanks a lot.
